src/App.js

```diff
@@ -17,8 +19,35 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
+    selectedTab: 0,
+  }
+
+  onTabSelected = (e) => {
+    this.setState({
+      selectedTab: tabs
+        .findIndex(tab => tab.title === e.target.textContent),
+    });
+  }
```

Comment:
you can pass index to this function

```suggestion
  onTabSelected = (index) => {
    this.setState({ selectedTabIndex: index });
  }
```